pep-0458.txt

```diff
@@ -471,12 +492,13 @@ __ https://github.com/theupdateframework/tuf/blob/v0.11.1/docs/TUTORIAL.md#deleg
 Based on our findings as of the time this document was updated for implementation
 (Oct 7 2019), PyPI SHOULD split all targets in the *bins* role by delegating
 them to 16,384 *bin-n* roles. Each *bin-n* role would sign for the PyPI targets whose
-hashes fall into that bin (see Figure 2).  It was found__
-that this number of bins would result in a 12-17% metadata overhead
+cryptographic hashes (specifically__ only SHA2-256) fall into that bin (see Figure 2).
```

Comment:
Why does "specifically" point to the metadata format section of the TUF spec?

Reply:
See one suggestion.